creusot-contracts/src/std/hint.rs

```diff
@@ -0,0 +1,35 @@
+use crate::*;
+
+extern_spec! {
+    mod std {
+        mod hint {
+            #[pure]
+            #[requires(cond)]
+            unsafe fn assert_unchecked(cond: bool) {}
+
+            #[pure]
+            #[ensures(true)]
```

Comment:
I was expecting `#[ensures(result == dummy)]` (similarly to `must_use` below). One reason I could see, is that this is the spec the compiler sees (in particular, it can't look at the implementation to figure out a better spec). But I'm not sure users expect this spec.

Reply:
Agreed. We should add a spec here.